src/Mailer/Event/SentMail.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Event;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+
+final class SentMail
```

Comment:
I think this should be `SendMail` and be in a command namespace rather than the event one. Since it's used as a command to actually send an email, and is handled by a commandhandler.